packages/zscaler_zia/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.20.1"
+  changes:
+    - description: Prevent failure on not encoded url`s
```

```suggestion
    - description: Prevent failure on not encoded URLs.
```
(sorry, I couldn't let this pass, but missed it before)